docs/application/dotnet/guides/multimedia/media-controller.md

```diff
@@ -121,7 +122,7 @@ var playlist = new MediaControlPlaylist("MyFavoriate",xx
 
```

Comment:
line 114. Is the xx ok?

var playlist = new MediaControlPlaylist("MyFavoriate",**xx**